code/src/main/java/com/google/sps/storage/data-storage/Place.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.storage;
+
+/**
+* Store a place's name and id
```

Comment:
meganit - let's try to use full sentences throughout. I won't block on that, but definitely for the future, and if you could do a rough pass now that'd be great.

Reply:
I think its done